### ISSUE TITLE
Dropdowns accessibility improvements

### ISSUE THREAD

**Issue by nstoychev** [telerik/kendo-react#996]
- Review the w3 wai-aria practices - https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html
- Consider to render hidden items when the dropdown is collapsed. We can render the list (or similar list) in the popup in a hidden span element when the dropdown is collapsed.
- Consider to remove role="listbox" of the root span. It could be change to aria-haspopup="listbox"
- handle reading of the default item

**Comment by simonssspirit** [comment]
reported again for the aria-owns connected list in ticket: 1532149

**Comment by shaizel** [comment]
I'm using the Kendo Angular DropDownList component and I'm facing the same problem, so it's not specific to kendo-react.
If this information should be moved somewhere else, let me know.

I've used Axe DevTools on the first example seen on https://www.telerik.com/kendo-angular-ui/components/dropdowns/dropdownlist/.

I got two issues, the same as those I get in my own application using the component and as were mentioned in [the forum post that raised this](https://www.telerik.com/forums/dropdownlist-fails-accessibility-requirement-wcag-1-3-1).
I've attached the output for both issues below.

First issue and [an explanation of the issue and its solutions](https://dequeuniversity.com/rules/axe/4.3/aria-input-field-name?application=AxeChrome):
![Kendo DropDownList accessibility issues 1](https://user-images.githubusercontent.com/25416352/143089704-86bb1ad8-f3d8-4051-af77-cbca3b402fb2.png)

Second issue and [an explanation of the issue and its solutions](https://dequeuniversity.com/rules/axe/4.3/aria-required-children?application=AxeChrome):
![Kendo DropDownList accessibility issues 2](https://user-images.githubusercontent.com/25416352/143089710-45a792de-946d-425f-a153-7313a73ac33f.png)

**Comment by simonssspirit** [comment]
@shaizel Please add this report to the Kendo Angular official repository, so the team can see it and plan it: https://github.com/telerik/kendo-angular/issues

**Comment by simonssspirit** [comment]
requested again with more details in 1539642

**Comment by tripathiakhil429** [comment]
@shaizel Could you please let us know if there is any ETA by when this issue will be fixed.

**Comment by simonssspirit** [comment]
Reported again in ticket 1546675

**Comment by InaGlushkova** [comment]
Changes are introduced with [v5.5.0](https://www.telerik.com/kendo-react-ui/components/changelogs/ui-for-react/#f41e9a37-40fc-55bf-b714-f3c8c7455a4e).

**Comment by srivai-sundarraj** [comment]
InaGlushkova 

Issue still reproducing.

I'm using kendo versions
`"@progress/kendo-angular-buttons": "^8.0.0",
    "@progress/kendo-angular-common": "^3.0.0",
    "@progress/kendo-angular-l10n": "^4.0.0",
    "@progress/kendo-angular-popup": "^5.0.0",
    "@progress/kendo-licensing": "^1.0.2",
    "@progress/kendo-theme-default": "^5.6.0",`


**Screens:**
![Screenshot from 2022-07-28 16-44-01](https://user-images.githubusercontent.com/479833/181493075-90986a8e-5b30-4597-9ca5-20ea6d9e7efc.png)
![Screenshot from 2022-07-28 16-43-47](https://user-images.githubusercontent.com/479833/181493083-6e0ae4d3-48fa-4f8a-b501-aeecd64b713e.png)

**Comment by nstoychev** [comment]
@srivai-sundarraj  Please add this report to the Kendo Angular official repository, so the team can see it and plan it: https://github.com/telerik/kendo-angular/issues

**Comment by EoinGriffin-AI** [comment]
Is this getting addressed? The current behaviour is still triggering AXE violations.

Here in the Telerik forums ( https://www.telerik.com/forums/dropdownlist-fails-accessibility-requirement-wcag-1-3-1 ) it was suggested to create this github issue as "a research task", and now I see this issue has been closed without linking to any work item.
So... it really looks like this has been brushed under the rug.

What's up with that?

**Comment by daphneSot** [comment]
Hello,

Is there a workaround for the aria owns issue? Is this issue fixed permanently?

Thanks
